trailblazer/server/api.py

```diff
@@ -63,6 +68,19 @@ def before_request():
         return abort(403, f"{user_data['email']} doesn't have access")
 
 
+@blueprint.route("/auth", methods=["POST"])
+@inject
+def authenticate(auth_service: AuthenticationService = Provide[Container.auth_service]):
+    try:
+        request_data = CodeExchangeRequest.model_validate(request.json)
```

Comment:
This is a bit unclear to me, where does the `request.json` come from?

Reply:
Me too! Read a little about the Flask library and apparently it makes the `request` object available in functions decorated with `@app.route` or `@blueprint.route`. It represents the incoming web request https://flask.palletsprojects.com/en/3.0.x/api/#flask.request.

I can't find any way of explicitly passing it, even if it could be nice to do so

Reply:
Ok, good to know. Thanks!